Header header
int32 leg_num
bool foot_off_ground
geometry_msgs/Point foot_position # Defined in body_ground frame
sensor_msgs/JointState joint_angle
